.github/workflows/release.yml

```diff
@@ -30,7 +30,7 @@ jobs:
 
       - id: release_info
         name: Get release information
-        run: python3 ci/release-info.py "$GITHUB_EVENT_NAME"
+        run: python3 ./ci/release-info-v3.py "$GITHUB_EVENT_NAME"
```

Comment:
Why create versioned scripts? We need just one.

Reply:
Replace `release-info.py` with this variant and drop the rest; we use version control to, well, control versions, and it's easier to track changes when the file gets modified in place.

Reply:
Thanks for mention that! Yeah, I will make only one script version, for sure, but I placed a three versions of scripts to show the differences between versions and use of different approaches, for preliminary review and for testing purposes, and for more simple changing scripts in [release.yml](https://github.com/input-output-hk/jormungandr/blob/master/.github/workflows/release.yml).